code/datums/brain_damage/phobia.dm

```diff
@@ -94,7 +94,7 @@
 		fearscore --
 	else if(stress && world.time > stress_check) //if we go long enough without fearing something, we begin to lose stress.
 		stress --
-		stress_check = world.time + min(3000, (600 * stress))
+		stress_check = world.time + min(1500, (300 * stress))
```

Comment:
no magic numbers

Reply:
bloons these are literally just deciseconds do you expect me to make a define for every decisecond in the game? this is not a proper place for defines and im also not refactoring phobias